content/docs/rfcs/24/README.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: 24
+title: 24/STATUS-CURATION
+name: Status Community Directory Curation Voting using Waku v2
+status: raw
+category: application
```

Comment:
Sorry, we have changed our mind on this one, it was in progress of being defined..

```suggestion
tags: waku-application
```